packages/adblocker-extended-selectors/src/eval.ts

```diff
@@ -71,104 +71,247 @@ export function matches(element: Element, selector: AST): boolean {
       }
 
       return text.length >= minLength;
+    } else if (selector.name === 'matches-path') {
+      const { argument } = selector;
+      if (argument === undefined) {
+        return false;
+      }
+
+      const path = globalThis.window.location.pathname;
+
+      let pattern = argument;
+      if (pattern.startsWith('/') && pattern.endsWith('/')) {
```

Comment:
I think you mentioned it wasn't used in any filters out there but in case there is any modifier like `i` (e.g. `/foo/i`) do we want to handle or maybe detect such cases in filters builder (or in unit tests)? Otherwise they would silently fail.